app/build.gradle.kts

```
@@ -59,6 +59,7 @@ android {
 }
 
 dependencies {
+    implementation(project(":feature:search:presentation"))
```

You can use implemented like:
```suggestion
    implementation(project(feature.search.presentation))
```